applications/StructuralMechanicsApplication/custom_conditions/surface_load_condition_3d.cpp

```diff
@@ -121,23 +122,21 @@ void SurfaceLoadCondition3D::CalculateAndSubKp(
     double coeff = 1.0;
     const std::size_t number_of_nodes = GetGeometry().size();
 
-    MakeCrossMatrix(cross_tangent_xi, rTangentXi);
-    MakeCrossMatrix(cross_tangent_eta, rTangentEta);
+    BeamMathUtils<double>::VectorToSkewSymmetricTensor(rTangentXi, cross_tangent_xi);
+    BeamMathUtils<double>::VectorToSkewSymmetricTensor(rTangentEta, cross_tangent_eta);
 
     for (std::size_t i = 0; i < number_of_nodes; ++i) {
-        const std::size_t RowIndex = i * 3;
+        const std::size_t row_index = i * 3;
         for (std::size_t j = 0; j < number_of_nodes; ++j) {
             const std::size_t column_index = j * 3;
 
-            coeff = Pressure * rN[i] * rDN_De(j, 1) * Weight;
-            noalias(Kij) = coeff * cross_tangent_xi;
-
             coeff = Pressure * rN[i] * rDN_De(j, 0) * Weight;
+            noalias(Kij) = coeff * cross_tangent_eta;
 
-            noalias(Kij) -= coeff * cross_tangent_eta;
+            coeff = Pressure * rN[i] * rDN_De(j, 1) * Weight;
+            noalias(Kij) -= coeff * cross_tangent_xi;
 
-            // NOTE TAKE CARE: the load correction matrix should be SUBTRACTED not added
-            MathUtils<double>::SubtractMatrix(rK, Kij, RowIndex, column_index);
+            MathUtils<double>::AddMatrix(rK, Kij, row_index, column_index);
```

Comment:
You sure abt this one?
The comment explicitly says not to do this

Reply:
I changed the order of the other ones